cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -28,7 +28,7 @@
 from pcluster.utils import get_installed_version, get_supported_os_for_architecture, get_supported_os_for_scheduler
 from pcluster.validators.common import FailureLevel, Validator
 
-NAME_MAX_LENGTH = 30
+NAME_MAX_LENGTH = 25
```

Comment:
This parameter is used by NameValidator.

NameValidator should be used for compute resource names and queues names but for shared storages we should replace NameValidator with SharedStorageNameValidator